Unix/tests/cli/test_cli.cpp

```diff
@@ -139,6 +139,64 @@ bool ParseCommandLine(MI_Char command[PAL_MAX_PATH_SIZE], vector<MI_Char*>& args
 #pragma prefast (pop)
 #endif
 
+/* =============================================================================
+**
+** Kill Old Server Instance
+**
+   ============================================================================*/
+
+/* We don't have a return value because this function will terminate the test 
+ * if it can't kill the previous omiserver. So success or program exit
+ */
+
+void KillOldServerInstance()
```

Comment:
Rather than having two instances, please try and grab the copy from util.c by linking against that.

Other than that, this looks fine. I'd prefer if it didn't hang for 2.5 seconds before recovering from a `kill -9` operation, but I understand your reluctance to change product code unnecessarily.